jasmine/private/jasmine_test.bzl

```diff
@@ -6,26 +6,36 @@ _attrs = dicts.add(js_binary_lib.attrs, {
     "entry_point": attr.label(
         mandatory = True,
     ),
+    "junit_reporter": attr.label(
+        mandatory = True,
+        allow_single_file = True,
+    ),
     "config": attr.label(
         doc = "jasmine config file. See: https://jasmine.github.io/setup/nodejs.html#configuration",
-        allow_single_file = [".json", ".js"]
-    )
+        allow_single_file = [".json", ".js"],
+    ),
 })
 
-
 def _impl(ctx):
-
     files = js_lib_helpers.gather_files_from_js_providers(
         targets = ctx.attr.data,
         include_transitive_sources = ctx.attr.include_transitive_sources,
         include_declarations = ctx.attr.include_declarations,
         include_npm_linked_packages = ctx.attr.include_npm_linked_packages,
     )
     files.extend(ctx.files.data)
+    files.append(ctx.file.junit_reporter)
+
+
+    junit_reporter = ctx.file.junit_reporter.short_path
+
+    if ctx.attr.chdir:
+        junit_reporter = "/".join([".." for _ in ctx.attr.chdir.split("/")] + [junit_reporter])
```

Comment:
Can `chdir` not have things like `.` or `..` in it? Can't we use something other then `.short_path`?

Reply:
short_part already gives us what we want. the pwd is `$RUNFILES_DIR/WKSP` and the short path is `../jasmine/junit_reporter.js`. chdir doesn't allow those characters as far as I know.

Reply:
I believe my options are either using a runfiles helper or processing this argument in runner.js like js_binary does in it's launcher. maybe @gregmagolan knows more?